cmd/provider/main.go

```diff
@@ -85,17 +86,22 @@ func main() {
 	})
 	kingpin.FatalIfError(err, "Cannot create controller manager")
 	kingpin.FatalIfError(apis.AddToScheme(mgr.GetScheme()), "Cannot add Azure APIs to scheme")
+
+	var runner terraform.ProviderRunner = terraform.NewNoOpProviderRunner()
+	if len(*nativeProviderPath) != 0 {
```

Comment:
A comment explaining that this is to enhance local development experience would be helpful.

Reply:
Thanks @muvaf. Added a comment explaining the motivation behind the new command-line flag `--terraform-native-provider-path`.